.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+*               @jellyfin/web
```

Comment:
So are we removing this for now or leaving it? I check all web PRs by default because I'm on the web team, so this seems redundant.

Reply:
I'm for removing it, it creates extra spam for me. I'm getting notified by GH anyway, but direct mentions or tags go to e-mail while others stay in GH notification only (in my setup).